test/m3u8.test.js

```diff
@@ -1,646 +1,650 @@
-(function(window, undefined) {
-  var
-    //manifestController = this.manifestController,
-    m3u8 = window.videojs.m3u8,
-    ParseStream = m3u8.ParseStream,
-    parseStream,
-    LineStream = m3u8.LineStream,
-    lineStream,
-    Parser = m3u8.Parser,
-    parser;
-
-  /*
-    M3U8 Test Suite
-  */
-
-  QUnit.module('LineStream', {
-    setup: function() {
-      lineStream = new LineStream();
+import {ParseStream, LineStream, Parser} from '../src/m3u8';
+import QUnit from 'qunit';
+import testDataExpected from './data/expected.js';
+import testDataManifests from './data/manifests.js';
+
+QUnit.module('LineStream', {
+  beforeEach() {
+    this.lineStream = new LineStream();
```

Comment:
Why not use module-level variables for these items? Technically the scope could leak across QUnit modules but it saves a ton of `this`s.